.gitignore

```diff
@@ -1,6 +1,7 @@
 *.py[cod]
 __pycache__
 .pytest_cache
+db.sqlite3
```

Comment:
This is added here, but there is a db.sqlite3 file that is going to be created with this PR, is that OK? @mariajgrimaldi